composer.json

```diff
@@ -12,11 +12,12 @@
     ],
     "require": {
         "php": ">=5.3",
-        "symfony/framework-bundle": "~2.3|~3.0",
-        "fzaninotto/faker": "~1.3"
+        "symfony/framework-bundle": "~2.3|~3.0|^4.0",
+        "fzaninotto/faker": "~1.3",
+        "phpunit/phpunit": "~4.8"
```

Comment:
This is not required. 

Generally, `phpunit/phpunit` should never be in the `require` section. And also, since we use `symfony/phpunit-bridge` we do not need `phpunit/phpunit` at all.